fitbenchmarking/results_processing/tables.py

```diff
@@ -65,12 +67,12 @@ def create_results_tables(options, results, group_name, group_dir, fig_dir,
     for suffix in SORTED_TABLE_NAMES:
         if suffix in options.table_type:
 
-            table_names[suffix] = \
-                f'{group_name}_{suffix}_{options.cost_func_type}_table.'
+            table_names[suffix] = f'{group_name}_{suffix}_table.'
```

Comment:
@tyronerees I seem to remember you saying something about filenames needing the cost function in. Obviously this doesn't work with multiple cost functions, so if it's needed for something can you let me know as we'll need to figure out a new solution...

Reply:
Ahh that makes sense. Does the problem directory not get wiped when it starts fresh anyway? Thanks for the clarification :)